src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -237,24 +244,76 @@ class SingleActiveFIMap extends React.Component<
     };
     const namePaths =
       plan.jurisdiction_name_path instanceof Array ? plan.jurisdiction_name_path : [];
-    const pages = namePaths.map(namePath =>
+    const pages = namePaths.map(namePath => {
       // return a page object for each name path
-      ({
+      return {
         label: namePath,
         url: '',
-      })
-    );
+      };
+    });
     breadCrumbProps.pages = [homePage, basePage, ...pages, secondLastPage];
+    const statusBadge =
```

Comment:
Can we have this render the value of `plan.plan_status` please?  I do not see a need to use hard coded strings here.